@xen-orchestra/backups/_cleanVm.js

```diff
@@ -445,18 +452,18 @@ exports.cleanVm = async function cleanVm(
     ...unusedVhdsDeletion,
     toMerge.length !== 0 && (merge ? Task.run({ name: 'merge' }, doMerge) : doMerge()),
     asyncMap(unusedXvas, path => {
-      onLog(`the XVA ${path} is unused`)
+      logWarn('unused XVA', { path })
       if (remove) {
-        onLog(`deleting unused XVA ${path}`)
+        logInfo('deleting unused XVA', { path })
         return handler.unlink(path)
       }
     }),
     asyncMap(xvaSums, path => {
       // no need to handle checksums for XVAs deleted by the script, they will be handled by `unlink()`
       if (!xvas.has(path.slice(0, -'.checksum'.length))) {
-        onLog(`the XVA checksum ${path} is unused`)
+        logInfo('unused XVA checksum', { path })
```

Comment:
it should be a warn a checksum without a xva is a potential problem
```suggestion
        logWarn('unused XVA checksum', { path })
```

Reply:
@julien-f tu en penses quoi ?

Reply:
We discussed it with @ByScripts and I do not consider this important enough to be a warning.

It only means that a checksum was not correctly deleted, I think an info is good enough in that case :slightly_smiling_face:

Reply:
ok for me , it's good enough with only a line in the syslog